testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -10792,27 +10760,30 @@ fn allow_reorg_within_first_proposal_burn_block_timing_secs_scenario() {
         return;
     }
 
+    pub const MINER1: usize = 1;
+    pub const MINER2: usize = 2;
+
     let num_signers = 5;
     let num_transfer_txs = 3;
 
     let test_context = Arc::new(SignerTestContext::new(num_signers, num_transfer_txs));
 
     scenario![
         test_context,
-        SkipCommitOpMiner2,
-        BootToEpoch3,
-        SkipCommitOpMiner1,
-        PauseStacksMining,
-        MineBitcoinBlock,
-        VerifyMiner1WonSortition,
-        SubmitBlockCommitMiner2,
-        ResumeStacksMining,
-        WaitForTenureChangeBlockFromMiner1,
-        MineBitcoinBlock,
-        VerifyMiner2WonSortition,
-        VerifyLastSortitionWinnerReorged,
-        WaitForTenureChangeBlockFromMiner2,
-        ShutdownMiners
+        (ChainMinerCommitOp::disable_for(test_context.clone(), MINER2)),
+        ChainBootToEpoch3,
+        (ChainMinerCommitOp::disable_for(test_context.clone(), MINER1)),
+        (ChainStacksMining::pause()),
+        (MinerMineBtcBlocks::one(test_context.clone())),
```

Comment:
Note for reviewers: this always mines one block, as per the expression

```rust
(MinerMineBtcBlocks::one(test_context.clone()))
```

madhouse uses the expression as-is.

---

If you pass a *type* instead (like `MineBitcoinBlock` as it was on the left side of the diff), it calls [`build`](https://github.com/stacks-network/stacks-core/pull/6161/files#diff-2c8ec86e739543da032494b8e5765ee78341a610c41fc392c578b346db7fba18R64), which currently chooses 2–5 blocks to mine.